message_ix/model/MESSAGE/data_load.gms

```diff
@@ -139,12 +139,7 @@ capacity_factor(node,tec,year_all2,year_all,'year') =
 operation_factor(node,tec,year_all2,year_all)$( map_tec(node,tec,year_all)
     AND map_tec_lifetime(node,tec,year_all2,year_all) AND NOT operation_factor(node,tec,year_all2,year_all) ) = 1 ;
 
-* set the addon-conversion factor to 1 by default
-addon_conversion(node,tec,vintage,year_all,mode,time,type_addon)$(
-    map_tec_addon(tec,type_addon)
-    AND map_tec_act(node,tec,year_all,mode,time)
-    AND map_tec_lifetime(node,tec,vintage,year_all)
-    AND NOT addon_conversion(node,tec,vintage,year_all,mode,time,type_addon) ) = 1 ;
+* addon-conversion factor is not set to 1 by default by default to avoid modes for technologies being set to 1 although they shouldn't
```

Comment:
@OFR-IIASA , "although they shouldn't" is really helpful to understand why this shouldn't be done

also, it would make more sense to insert this in the `parameter_def.gms` file where all parameters are described, rather than having a comment in `data_load.gms` that something is not done

Reply:
@danielhuppmann done